src/test/java/bio/terra/app/controller/StatusTest.java

```diff
@@ -32,6 +36,14 @@
     private MockMvc mvc;
     @Autowired
     private ConfigurationService configurationService;
+    @Autowired
+    @Qualifier(SPRING_CACHE_MANAGER)
+    private CacheManager cacheManager;
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Should the "u" be lowercase? setup()

Reply:
Ah, I'm used to the capital U.  Happy to lowercase

Reply:
Actually, it probably doesn't matter since we won't override it anywhere.